### ISSUE TITLE
Handling institutional authors

### ISSUE THREAD

**Issue by supersambo** [papis/papis#885]
* papis, version 0.13:

I am having trouble dealing with institutional authors. In order for them to be cited correctly using biblatex they must be exported to bib-files as such: `{{The name of the institutions}}`. Otherwise this gets interpreted as name-surname-pair and results in something like "Name of the institution, T." when referenced.

# how to reproduce the issue
Consider the following bib file `ipcc1990.bib`
```
@report{ipcc1990,
  author={{Intergovernmental Panel on Climate Change}},
  title={Contribution of Working Group I to the First Assessment Report of the Intergovernmental Panel on Climate Change},
  year={1990},
}
```
After importing it via `papis add --from bibtex ipcc1990.bib` it looks something like this:

``` 
author: on Climate Change, Intergovernmental Panel
author_list:
- family: on Climate Change
  given: Intergovernmental Panel
files: []
papis_id: ac0ce27201d0360e806daa3924f77201
ref: ipcc1990
time-added: 2024-07-10-11:23:44
title: Contribution of Working Group I to the First Assessment Report of the Intergovernmental Panel on Climate Change
type: report
year: '1990'

```
The author gets interpreted as name-surname-pair. I could live with that and change it manually to:
```
author: {{Intergovernmental Panel on Climate Change}}
files: []
papis_id: ac0ce27201d0360e806daa3924f77201
ref: ipcc1990
time-added: 2024-07-10-11:23:44
title: Contribution of Working Group I to the First Assessment Report of the Intergovernmental Panel on Climate Change
type: report
year: '1990'
```
Unfortunately, when trying to export that I get an error:

```sh
$ papis export ipcc1990 -f bibtex
[INFO] database.cache: Indexing library. This might take a while...
[ERROR] document: Error reading info file at '~/papis/ipcc1990__ac0ce27201d0360e806daa3924f77201/info.yaml'. Please check it! (Caught exception 'ValueError: while constructing a mapping
  in...'. Use `--log DEBUG` to see traceback)
No document selected!
```

For some reason, this also deletes all metadata from the yaml file except for the id:

```sh
$ cat ~/papis/ipcc1990__ac0ce27201d0360e806daa3924f77201/info.yaml
papis_id: e1f27a34adca7f3764a44fe99b5dcf3a
```
Any idea on how to deal with this? For now, any type of dirty hack would also be much appreciated!

**Comment by supersambo** [comment]
Found the dirty hack: Putting the double curly brackets in parenthesis seems to work

```yaml
author: '{{Intergovernmental Panel on Climate Change}}'
files: []
papis_id: ac0ce27201d0360e806daa3924f77201
ref: ipcc1990
time-added: 2024-07-10-11:23:44
title: Contribution of Working Group I to the First Assessment Report of the Intergovernmental Panel on Climate Change
type: report
year: '1990'`

```

**Comment by alexfikl** [comment]
@supersambo There's an attempt at a proper fix in #886. Can you try if that works for you?

**Comment by alexfikl** [comment]
@supersambo I'll go ahead and merge #886 as soon as the CI passes. Feel free to re-open this if you find some more failing edge cases!
